modules/ROOT/pages/type-definitions/directives/genai.adoc

```diff
@@ -0,0 +1,170 @@
+:description: Directives related to generative AI in the Neo4j GraphQL Library.
+
+[[type-definitions-genai]]
+= Generative AI
+
+With the GraphQL directives for generative AI you can query your database for vector indexes.
+Queries are either performed by the vector index of a specific node or by a query phrase.
+
+A query by vector index finds nodes with a vector embedding similar to that index, which is also referred to as nearest neighbor search.
+
+In contrast, a query by phrase (a string of text) passes the phrase to the link::https://neo4j.com/docs/cypher-manual/current/genai-integrations/[Neo4j GenAI plugin] and the plugin generates a vector embedding for it.
+This embedding is then compared to the node vector embeddings in the database and the result set 
+
+[NOTE] 
+.Prerequisites
+==== 
+* The node vector embeddings already exist
+* The embeddings must have been created using the same model
```

Comment:
By model you mean method?

Reply:
good catch. i guess "method" is clearer, yes.